Gemfile.travis

```diff
@@ -1,6 +1,6 @@
 source "http://rubygems.org"
 
-gem "straight", git: "https://github.com/snitko/straight.git"
+gem "straight", github: "AlexanderPavlenko/straight", ref: "d5a05d7ea1722146aaf48a8b91e4c6eeb73ff007"
```

Comment:
Sure, but in this case, it would have been better to let it fail and wait for me to merge PR for `straight`.